packages/jaeger-ui/src/components/DeepDependencies/Header/LayoutSettings/index.test.js

```diff
@@ -21,9 +21,11 @@ import * as track from '../../index.track';
 
 import { EDdgDensity } from '../../../../model/ddg/types';
 
+const defaultDensityFromLocalStorage = localStorage.getItem('selectedDensity');
+
 describe('LayoutSettings', () => {
   const props = {
-    density: EDdgDensity.PreventPathEntanglement,
+    density: defaultDensityFromLocalStorage || EDdgDensity.PreventPathEntanglement,
```

Comment:
don't see why you need this

Reply:
Yes, I was wrong it's working fine without this additional configuration. I have resolved it back to it's original state